ui/tests/acceptance/client-detail-test.js

```diff
@@ -288,9 +286,27 @@ test('/clients/:id should list all attributes for the node', function(assert) {
   });
 });
 
-test('when the node is not found, an error message is shown, but the URL persists', function(
-  assert
-) {
+test('/clients/:id lists all meta attributes', function(assert) {
+  node = server.create('node', 'forceIPv4', 'withMeta');
+
+  visit(`/clients/${node.id}`);
+
+  andThen(() => {
+    assert.ok(find('[data-test-meta]'), 'Meta attributes table is on the page');
+    assert.notOk(find('[data-test-empty-meta-message]'), 'Meta attributes is not empty');
```

Comment:
Perhaps an assertion about the contents of the table here, to ensure that the correct attributes are bound through?

Reply:
Totally. I skipped this time (and last time) because there are already integration tests for the component, but you're right that asserting what data is being bound is different and untested.